crates/polars-core/src/chunked_array/ops/flarion_funcs.rs

```diff
@@ -43,7 +43,8 @@ pub fn flarion_substring(s: &str, from: i32, len: i32) -> String {
         let mut found_start = false;
 
         let distance_to_advance = from.unsigned_abs() as usize;
-        let end_idx = std::cmp::max(0, distance_to_advance as i32 - len) as usize; // We know len is positive
+
+        let end_idx = (distance_to_advance as i32).wrapping_sub(len) as usize; // We know len is positive
```

Comment:
Wrapping sub guarantee proper indexing?

Reply:
wrapping sub is the same as before but without panic

Reply:
i was wrong, it's not working

Reply:
fixed